### DISCUSSION TITLE
Recommend other artists on your artist page

### DISCUSSION THREAD

**Discussion by Cwpute** [funmusicplace/mirlo#715] [Ideas]
Being able to add a section on your artist/user page to recommend other artists, either on Mirlo or elsewhere.

This will help users find artists that potentially do the same thing, or that know each other. It can help a local community get the word out about their neighbor's music, or put the light on artists you're following yourself and believe are doing interesting stuff with their music.

Before any (*if* any) recommendation system is put into place, this will allow the community to naturally create interactions with one another, and can potentially help gather data about "influential artists" that often get recommended.

**Comment by LouisLouiseKay** [comment]
Strong agree on this, I thought about is last week when using Bandcamp. This is a mix of serendipity and trusting creators to build a sort of webring curration system and it's an extremely good and unharmful way to help with discovery.

**Comment by simonv3** [comment]
For what it's worth we've implmented recommended albums from album pages. Though not yet recommended artists.

**Reply by Cwpute** [reply]
While recommended albums are great ( :tada: ) being able to recommend artists will help those recommended control what they show to new people coming from the recommendation section - while album recommendations only funnels them through potentially dated releases they don't want publicity for anymore.
On the other hand if an artist is particularly fan of a specific album from another artist's discography, album recommendations allow them to be more precise.
So both recommendation types complement each other.
